openssl/src/dh.rs

```diff
@@ -9,10 +23,17 @@ use bn::BigNum;
 
 foreign_type! {
     type CType = ffi::DH;
+
     fn drop = ffi::DH_free;
 
+    /// Diffie-Hellman Key agreement object
+    /// A key object that is based on the diffie-hellman algorithm
+    ///
+    /// As described here https://wiki.openssl.org/index.php/Manual:Dh(3)
+    /// and here: https://wiki.openssl.org/index.php/Diffie_Hellman
     pub struct Dh;
 
+    /// a borrowed reference of the dh type, diffie-hellman key agreement object
```

Comment:
Capitalization, punctuation.

I like the format [here](https://github.com/sfackler/rust-openssl/pull/743/files#diff-3399783e0db3cafe8354809089223aaeR57) for documentation for the `*Ref` types.

Reply:
Does doing something like [Dh] in RustDoc link to the type? I didn't know about this feature at all!

Reply:
The `[Foo]` part is a reference to the link defined as `[Foo]: https://foo.com` below.